frontend/i18next.scanner/fr/translation.json

```diff
@@ -1,4 +1,12 @@
 {
+  " at ": " dans ",
+  " added you ": " vous a ajouté ",
+  " created ": " a créé ",
+  " commented on ": " a commenté ",
+  " mentioned you in ": " vous a mentionné dans ",
+  " mentioned you in a comment in ": " vous a mentionné dans un commentaire sur ",
+  " updated a new version of ": " a mis à jour nouvelle version de ",
+  " updated the status of ": " a mis à jour le statut de ",
```

Comment:
since the name of the document and the shared space have a different design from the line of the sentence, was the way I found to get the html tags, would you have another idea?